lib/jsonapi/view.ex

```diff
@@ -94,23 +96,38 @@ defmodule JSONAPI.View do
         do: serialize(__MODULE__, models, conn)
 
       def url_for(nil, nil) do
-        "#{@namespace}/#{type()}"
+        case @host do
+        ""   -> "#{@namespace}/#{type()}"
+        host -> "#{host}#{@namespace}/#{type()}"
+        end
       end
 
       def url_for(data, nil) when is_list(data) do
-        "#{@namespace}/#{type()}"
+        case @host do
```

Comment:
You know Im not completely sure. I had noticed that namespace was being used and thought putting the host made sense at the time, however I do agree that if there is no connection there isn't a need for a host and possibly a namespace as well. Is there a preferred behavior we want?